ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -369,6 +369,22 @@ let assert_gpus_available ~__context ~self ~host =
         Ref.string_of (List.hd not_available)
       ]))
 
+let assert_usbs_available ~__context ~self ~host =
+  let vusbs = Db.VM.get_VUSBs ~__context ~self in
```

Comment:
Minor point: but you can write this as:
```
Db.VM.get_VUSBs ~__context ~self
|> List.iter (fun vusb -> ..
```
This has the benefit that that data flows from top to bottom.